libs/labelbox/tests/integration/test_model_config.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+def test_create_model_config(client, valid_model_id):
+    model_config = client.create_model_config("model_config", valid_model_id, {"param": "value"})
+    assert model_config.inference_params["param"] == "value"
+    assert model_config.name == "model_config"
+    assert model_config.model_id == valid_model_id
+
+
+def test_delete_model_config(client, valid_model_id):
+    model_config_id = client.create_model_config("model_config", valid_model_id, {"param": "value"})
+    assert(client.delete_model_config(model_config_id.uid))
```

Comment:
is there a way to retrieve it back and make sure it does not exist

Reply:
I'll add one.